app/sprinkles/account/config/default.php

```diff
@@ -113,7 +113,14 @@
                     'user' => true
                 ]
             ]
-        ]
+        ],
+        'password_security' => [
+             'enforce_no_compromised'   => '5', // Set to false to turn off this feature. Otherwise, provide a numeric string, which sets the maximum number
+                                       // of times that is "acceptable" for a password to have appeared in breaches. The recommended and most secure
+                                       // option is '0' - meaning only passwords that are not on the list of compromised passwords will be allowed.
+
+             'enforce_update_passwords' => true // Settings this to true will check user's passwords against list of known compromised passwords at each log on.
```

Comment:
I think this would better belong under the login group above. Comment is important, however semantically this isn't where you would expect a login specific setting to reside. At least that is my take on it.

Reply:
I think you are right and logically `enforce_update_passwords` would be in the login group. However, `enforce_no_compromised` is not necessarily a login specific setting... 

Any suggestions on better key names for these two items?

Reply:
I'm thinking something along the lines of having the `password_security` group define criteria passwords must adhere to (e.g. complexity customisation, exposures, etc) for the sake of extend-ability in the future. So basically the rest stays as is.

Reply:
I changed `enforce_update_passwords` to `enforce_reset_compromised` and moved it under the login group. Although `update` would work, `reset` is more in alignment with the existing verbiage.